### PR TITLE
Display tables correctly

### PR DESCRIPTION
**Pull request by Siedlerchr**
Added github table css

### CODE REVIEW

**Comment by Siedlerchr** [comment]
Hm, no idea why there is still a conflict

**Comment by Siedlerchr** [comment]
I tested the changes locally with a jekyll installation. The tables render now the same as in the github preview.

File: en/SearchHelp.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Searching
-outdated:true
+outdated: true
```
Comment (Siedlerchr) [inline en/SearchHelp.md]: jekyll complained about the missing space.

**Comment by Siedlerchr** [comment]
@koppor  Please have a look

File: _config.yml [hunk 2]
```diff
@@ -15,8 +15,16 @@ lsi: false
 #pygments is not supported by GitHub. See https://help.github.com/articles/page-build-failed-config-file-error/#fixing-highlighting-errors
 #See also https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-jekyll-3-0
 #We have to use rouge
+#http://idratherbewriting.com/2016/02/21/bug-with-kramdown-and-rouge-with-github-pages/
 highlighter: rouge
 markdown: kramdown
+kramdown:
+  input: GFM
+  auto_ids: true
```
Comment (Siedlerchr) [inline _config.yml]: This enables possibility to autogeneerate toc

File: _config.yml [hunk 3]
```diff
@@ -15,8 +15,16 @@ lsi: false
...
+  syntax_highlighter: rouge
+
+sass:
+  sass_dir: _sass
```
Comment (koppor) [inline _config.yml]: kudos for _sass

Do you have [openhub](https://www.openhub.net/p/jabref)? :)

Comment (Siedlerchr) [inline _config.yml]: Yep, already registered https://www.openhub.net/accounts/siedlerchr

**Pull request merged**